src/apps/goldfinch/ethereum/goldfinch.staking-rewards.contract-position-fetcher.ts

```diff
@@ -0,0 +1,178 @@
+    return [
+      {
+        address: STAKING_REWARDS,
+        underlyingTokenAddress: FIDU,
+      },
+      {
+        address: STAKING_REWARDS,
+        underlyingTokenAddress: CURVE_FIDU_USDC,
+      },
+      {
+        address: STAKING_REWARDS,
+        underlyingTokenAddress: GFI,
+      },
+    ];
+  }
+
+  async getTokenDefinitions({
+    definition,
+  }: GetTokenDefinitionsParams<GoldfinchStakingRewards, GoldfinchStakingRewardsDefinition>) {
+    const GFI = '0xdab396ccf3d84cf2d07c4454e10c8a6f5b008d2b';
+    return [
+      {
+        metaType: MetaType.SUPPLIED,
+        address: definition.underlyingTokenAddress,
+        network: Network.ETHEREUM_MAINNET,
+      },
+      {
+        metaType: MetaType.CLAIMABLE,
+        address: GFI,
+        network: Network.ETHEREUM_MAINNET,
+      },
+    ];
+  }
+
+  async getDataProps() {
+    return {
+      assetStandard: Standard.ERC_721,
+    };
+  }
+
+  async getLabel({ contractPosition }: GetDisplayPropsParams<GoldfinchStakingRewards>) {
+    return `Staked ${getLabelFromToken(contractPosition.tokens[0])}`;
+  }
+
+  async getTokenBalancesPerPosition(): Promise<BigNumberish[]> {
+    throw new Error('Method not implemented.');
+  }
+
+  async getBalances(address: string): Promise<ContractPositionBalance<GoldfinchStakingRewardsDataProps>[]> {
+    const GFI = '0xdab396ccf3d84cf2d07c4454e10c8a6f5b008d2b';
+    const FIDU = '0x6a445e9f40e0b97c92d0b8a3366cef1d67f700bf';
+    const CURVE_FIDU_USDC = '0x42ec68ca5c2c80036044f3eead675447ab3a8065';
+    const multicall = this.appToolkit.getMulticall(this.network);
+    const positions = await this.appToolkit.getAppContractPositions<GoldfinchStakingRewardsDataProps>({
+      appId: this.appId,
+      groupIds: [this.groupId],
+      network: this.network,
+    });
+
+    const fiduPosition = positions.find(v => v.tokens[0].address === FIDU);
+    const curveFiduUsdcPosition = positions.find(v => v.tokens[0].address === CURVE_FIDU_USDC);
+    const gfiPosition = positions.find(v => v.tokens[0].address === GFI);
+
+    if (!fiduPosition || !curveFiduUsdcPosition || !gfiPosition) return [];
+
+    const stakingRewardsContract = this.contractFactory.goldfinchStakingRewards({
+      address: fiduPosition.address,
+      network: this.network,
+    });
+    const balanceRaw = await multicall.wrap(stakingRewardsContract).balanceOf(address);
+    const balance = Number(balanceRaw);
+    if (balance === 0) return [];
+
+    const claimableGFI = await multicall.wrap(stakingRewardsContract).totalOptimisticClaimable(address);
+    const claimableGFITokens = [drillBalance(gfiPosition.tokens[1], claimableGFI.toString())];
+    const claimableGFIBalanceUSD = sumBy(claimableGFITokens, v => v.balanceUSD);
+    const claimableGFIContractPositionBalance = {
+      ...gfiPosition,
+      tokens: claimableGFITokens,
+      balanceUSD: claimableGFIBalanceUSD,
+    };
+
+    let sumOfCurveTotalTokens = new BigNumber(0);
```

Comment:
this is also new, instead of showing 4 different positions - I am summing them up all into one. 

<img width="823" alt="Screenshot 2023-01-17 at 7 27 58 PM" src="https://user-images.githubusercontent.com/6665100/213076589-1c67bf6c-07f4-45d0-840c-df6b457f7c1d.png">